src/Bundle/Twig/Token/WebpackTokenParser.php

```diff
@@ -26,13 +26,12 @@ class WebpackTokenParser implements \Twig_TokenParserInterface
     private $extension;
 
     /**
-     * @var int
+     * @var int[]
```

Comment:
[existing] Some docs on possible values of this variable would be much appreciated